ZuluContent/Spells/Base/SpellHelper.cs

```diff
@@ -190,6 +184,38 @@ public static bool AdjustField(ref Point3D p, Map map, int height, bool mobsBloc
             return false;
         }
 
+        public static double CalcSpellDamage(Mobile caster, Mobile defender, Spell spell, bool areaSpell = false)
+        {
+            const int mageryDivider = 5;
+            const int onPlayerDivider = 3;
```

Comment:
Maybe rather call this playerDivider and reserve anything "on" to be for event handlers

Reply:
Yeah good point, I carried over the name from the eScript to help with cross-checking, it's `SPELL_DAMAGES_ON_PLAYER_DIVIDER` in spelldata.inc